README.rst

```diff
@@ -167,6 +167,12 @@ working directory.  You can specify a different file via the
 
     pytest --docker-compose=/path/to/docker-compose.yml
 
+To specify more than one compose file, separate them with a ``,``:
```

Comment:
❔ Could this be expanded a bit, to explain the result of specifying multiple compose files, and maybe when it could be useful to do this?

Reply:
I agree it's a relatively unknown feature (I think?) So a quick link to the docker-compose documentation where it explains how multiple files work is definitely nice!
https://docs.docker.com/compose/extends/